app/controllers/organizations/membership_requests_controller.rb

```diff
@@ -9,6 +9,10 @@ def index
   def approve
     authorize Membership, :create?
     @membership_request.approve!(completed_by: current_user)
+    recipient = @membership_request.user
+    if recipient
+      MembershipRequestAcceptedNotifier.with(organization: @organization).deliver(recipient)
+    end
```

Comment:
maybe we can move this too into the model? I think here
https://github.com/yshmarov/moneygun/blob/d4d9b7e60ca609ed3720778bd6e54b17db7678b3/app/models/membership_invitation.rb#L31